<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerTextActionsMenu</class>
 <widget class="QWidget" name="ViewerTextActionsMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>168</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>792</width>
    <height>168</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true" />
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="arrowCont2" native="true">
     <widget class="QLabel" name="topArrowLbl">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>3</y>
        <width>34</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string />
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="horizontalWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="popupCont">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>14142</width>
          <height>113</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="notesActionBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Crear nota</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteActionBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Borrar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="underlineActionBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Subrayar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="defineActionBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Definir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchActionBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Buscar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="wikipediaMenuBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string />
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="arrowCont" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>27</height>
      </size>
     </property>
     <widget class="QLabel" name="bottomArrowLbl">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>34</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string />
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>28</width>
       <height>57</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources />
 <connections />
</ui>